src/Controls/tests/DeviceTests/ControlsHandlerTestBase.cs

```diff
@@ -183,9 +184,16 @@ protected Task CreateHandlerAndAddToWindow<THandler>(IElement view, Func<THandle
 						else
 							throw new Exception($"I can't work with {typeof(THandler)}");
 
-						window.Deactivated();
-						window.Destroying();
 
+						bool isActivated = (window as Window)?.IsActivated ?? false;
+						bool isDestroyed = (window as Window)?.IsDestroyed ?? false;
```

Comment:
Are this properties part of the IWindow interface?

Reply:
No, these properties are currently specific to the `Controls` layer (non-interface layer). At a later date we could make these properties of `IWindow` but we are going to backport this PR so don't want to make any API changes here.